frontend/components/inbox/TicketOverview.vue

```diff
@@ -130,6 +132,12 @@ export default {
         this.tickets = response.data
       })
     },
+    callDebounceSearch:_.debounce(function(){
+      this.debounceSearch();
+    }, 300),
```

Comment:
je kunt dit als het goed is vervangen door callDebounceSearch: _.debounce(this.get_tickets, 300)